dataclients/kubernetes/ingress.go

```diff
@@ -490,11 +485,11 @@ func (ing *ingress) addSpecRule(ic ingressContext, ru *definitions.Rule) error {
 	// it is a regexp, would be better to have exact host, needs to be added in skipper
 	// this wrapping is temporary and escaping is not the right thing to do
 	// currently handled as mandatory
-	host := []string{"^" + strings.Replace(ru.Host, ".", "[.]", -1) + "$"}
+	host := []string{createHostRx(ru.Host)}
 	// update Traffic field for each backend
 	computeBackendWeights(ic.backendWeights, ru)
 	for _, prule := range ru.Http.Paths {
-		addExtraRoutes(ic, host, ru.Host, prule.Path, ing.eastWestDomainRegexpPostfix, ing.kubernetesEnableEastWest)
+		addExtraRoutes(ic, host, ru.Host, prule.Path, ing.kubernetesEastWestDomain, ing.kubernetesEnableEastWest)
```

Comment:
Since we change here, lets remove
https://github.com/zalando/skipper/blob/c518cb137eaaa8f4453f9904d13f6d5352527363/dataclients/kubernetes/ingress.go#L490-L493
pass only `ru.Host` and create regexp inside of `addExtraRoutes`
https://github.com/zalando/skipper/blob/c518cb137eaaa8f4453f9904d13f6d5352527363/dataclients/kubernetes/ingress.go#L378